addons/mail/static/src/js/discuss.js

```diff
@@ -933,12 +928,14 @@ var Discuss = Widget.extend(ControlPanelMixin, {
     },
     /**
      * @private
+     * @param {MouseEvent} event
      */
-    _onSettingsButtonClicked: function () {
+    _onSettingsChannel: function (event) {
```

Comment:
Shouldn't it be _onSettingsChannelClicked ? I woud like not to hurt @ged-odoo feelings.

Reply:
yep, _onSettingsChannelClicked is better

Reply:
It's not very clear here but I tried to match other functions declarations. 
_onSettingsButtonClicked was for a top button (no event), 
_onSettingsChannel id for channel button (with event) , like _onUnpinChannel. 

But I agree that _onSettingsChannelClicked would be more correct can change that

Reply:
Unpin is actually a verb.  The idea is that we try to express the fact that an event just happened, but we are definitely not 100% consistend

Reply:
You are totally right.

Reply:
Wouldn't _onChannelSettingsClicked be even more appropriate?